### PR TITLE
Add isort check for changed files

### PR DESCRIPTION
**Pull request by fernflower**
After discussing with pstodulk a massive change introducing isort
rules and fixing all occurrences like https://github.com/oamg/leapp-repository/pull/770 may be too annoying, so let's
make it gradually - intoduce a handmade isort check for changed files
only.

### CODE REVIEW

**Comment by fernflower** [comment]
Works like a charm :)
```
 ------------------------------------
Your code has been rated at 10.00/10

--- Running flake8 ---
if [[ "${PYTHON_VENV:-python2.7}" == "python2.7" ]] ; then \
	. tut/bin/activate; \
	echo "--- Checking py3 compatibility ---" && \
	SEARCH_PATH=repos && \
	bash -c "[[ ! -z '${SEARCH_PATH}' ]] && find ${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint --py3k" && \
	echo "--- Linting done. ---"; \
fi
. tut/bin/activate; \
git diff master..HEAD --name-only | xargs isort -c --diff && \
echo "--- isort check done. ---;"
ERROR: /payload/repos/system_upgrade/el7toel8/actors/postgresqlcheck/actor.py Imports are incorrectly sorted and/or formatted.
--- /payload/repos/system_upgrade/el7toel8/actors/postgresqlcheck/actor.py:before	2021-11-30 19:27:03.485523
+++ /payload/repos/system_upgrade/el7toel8/actors/postgresqlcheck/actor.py:after	2021-11-30 19:28:49.677177
@@ -1,7 +1,7 @@
 from leapp.actors import Actor
 from leapp.libraries.actor.postgresqlcheck import report_installed_packages
 from leapp.models import InstalledRedHatSignedRPM, Report
-from leapp.tags import IPUWorkflowTag, ChecksPhaseTag
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
 
 
 class PostgresqlCheck(Actor):
make: *** [Makefile:248: lint] Error 123
Script done, file is typescript
```
@pirat89 @vinzenz @Rezney @MichalHe  How do you like this gradual imports sorting rule introduction?

File: Makefile [hunk 1]
```diff
@@ -259,6 +259,9 @@ lint:
 		bash -c "[[ ! -z '$${SEARCH_PATH}' ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint --py3k" && \
 		echo "--- Linting done. ---"; \
 	fi
+	. $(VENVNAME)/bin/activate; \
+	git diff $(MASTER_BRANCH)..HEAD --name-only | xargs isort -c --diff && \
+	echo "--- isort check done. ---;"
```
Comment (shaded-enmity) [inline Makefile:264]: I like where this is going - here's an untested suggestion that will automatically apply the generated patch using `git apply` and then use `git diff` to print the changes.

```suggestion
	git diff $(MASTER_BRANCH)..HEAD --name-only | xargs isort -c --diff | git apply - && git diff --exit-code && \
	echo "--- isort check done. ---;" || echo "--- isort check done - new changes staged. ---;"
```

Comment (fernflower) [reply Makefile:264]: Well hypothetically isort auto-applied changes can introduce syntax errors, so I would leave the actual fixing up to the authors (they can run isort themselves and verify the changes). I'd leave linter it's usual role by default - inform about the problem, not fix it.

**Review by vinzenz — APPROVED** [review]
LGTM

**Comment by shaded-enmity** [comment]
@fernflower I'd love to see more automated code style enforcement like that in the future - at least for me this was the most boring and tedious part of reviews - perhaps `make lint_fix` as more general "placeholder" for futures such changes - so that we can use the current target for testing and the `lint_fix` target to attempt to automatically resolve found problems.

**Review by shaded-enmity — APPROVED** [review]
LGTM! :1st_place_medal:

**Comment by fernflower** [comment]
Will put there how a list of violation looks like on some currently open pr like https://github.com/oamg/leapp-repository/pull/749
```
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
index e0ae41b..7eca815 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
@@ -3,11 +3,11 @@ import shutil
 
 from leapp import reporting
 from leapp.actors import Actor
-from leapp.models import SELinuxModules, SELinuxCustom
-from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
 from leapp.libraries.actor import selinuxapplycustom
 from leapp.libraries.actor.selinuxapplycustom import BACKUP_DIRECTORY
-from leapp.libraries.stdlib import run, CalledProcessError
+from leapp.libraries.stdlib import CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxModules
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
 
 WORKING_DIRECTORY = '/tmp/selinux/'
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
index 95c06bf..887f381 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
@@ -1,9 +1,8 @@
-import re
 import os
+import re
 import shutil
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
-
+from leapp.libraries.stdlib import api, CalledProcessError, run
 
 BACKUP_DIRECTORY = "/tmp/leapp-selinux/"
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
index b0ed347..0171f58 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
@@ -2,8 +2,8 @@ import os
 
 import pytest
 
-from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxFacts, SELinuxModule, SELinuxModules, SELinuxRequestRPMs
 from leapp.reporting import Report
 
 TEST_MODULES = [
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
index b764a51..23bc9c2 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
@@ -1,7 +1,7 @@
 from leapp.actors import Actor
-from leapp.models import SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs, RpmTransactionTasks
-from leapp.tags import FactsPhaseTag, IPUWorkflowTag
 from leapp.libraries.actor import selinuxcontentscanner
+from leapp.models import RpmTransactionTasks, SELinuxCustom, SELinuxFacts, SELinuxModules, SELinuxRequestRPMs
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
 
 
 class SELinuxContentScanner(Actor):
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
index e2f3de8..d512803 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
@@ -2,8 +2,8 @@ import os
 import re
 from shutil import rmtree
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
 from leapp.libraries.common.config import version
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule
 
 # types and attributes that where removed between RHEL 7 and 8
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
index 2a98aa4..956277b 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
@@ -2,11 +2,11 @@ import os
 
 import pytest
 
-from leapp.snactor.fixture import current_actor_context
-from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
 from leapp.libraries.common.config import mock_configs
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxFacts, SELinuxModule, SELinuxModules, SELinuxRequestRPMs
 from leapp.reporting import Report
+from leapp.snactor.fixture import current_actor_context
 
 # compat module ensures compatibility with newer systems and is not part of testing
 TEST_MODULES = [
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
index 0ab2623..96e49b4 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
@@ -1,4 +1,4 @@
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModules
 
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
index 078ee76..105dcb2 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
@@ -2,12 +2,11 @@ import os
 
 import pytest
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule, SELinuxModules
 from leapp.reporting import Report
 from leapp.snactor.fixture import current_actor_context
 
-
 TEST_MODULES = [
     ["400", "mock1"],
     ["99", "mock1"],
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
index beeaf3e..0a6f4c9 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
@@ -1,5 +1,5 @@
 from leapp.libraries.actor import selinuxprepare
-from leapp.libraries.stdlib import CalledProcessError, api, run
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule, SELinuxModules
 
 
diff --git a/repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py b/repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py
index 7094a75..b173f0e 100644
--- a/repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py
+++ b/repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py
@@ -1,12 +1,20 @@
+import functools
 import os
 import subprocess
-import functools
-
 
-from leapp.models import StorageInfo, PartitionEntry, FstabEntry, MountEntry, LsblkEntry, \
-    PvsEntry, VgsEntry, LvdisplayEntry, SystemdMountEntry
 from leapp import reporting
 from leapp.libraries.stdlib import api
+from leapp.models import (
+    FstabEntry,
+    LsblkEntry,
+    LvdisplayEntry,
+    MountEntry,
+    PartitionEntry,
+    PvsEntry,
+    StorageInfo,
+    SystemdMountEntry,
+    VgsEntry
+)
 
 
 def aslist(f):
diff --git a/repos/system_upgrade/common/actors/storagescanner/tests/unit_test_storagescanner.py b/repos/system_upgrade/common/actors/storagescanner/tests/unit_test_storagescanner.py
index 3d942ec..7ee0c7f 100644
--- a/repos/system_upgrade/common/actors/storagescanner/tests/unit_test_storagescanner.py
+++ b/repos/system_upgrade/common/actors/storagescanner/tests/unit_test_storagescanner.py
@@ -4,8 +4,16 @@ from leapp import reporting
 from leapp.libraries.actor import storagescanner
 from leapp.libraries.common.testutils import create_report_mocked, logger_mocked
 from leapp.libraries.stdlib import api
-from leapp.models import (FstabEntry, LsblkEntry, LvdisplayEntry, MountEntry, PartitionEntry, PvsEntry,
-                          SystemdMountEntry, VgsEntry, )
+from leapp.models import (
+    FstabEntry,
+    LsblkEntry,
+    LvdisplayEntry,
+    MountEntry,
+    PartitionEntry,
+    PvsEntry,
+    SystemdMountEntry,
+    VgsEntry
+)
 
 CUR_DIR = os.path.dirname(os.path.abspath(__file__))
 
diff --git a/repos/system_upgrade/common/libraries/utils.py b/repos/system_upgrade/common/libraries/utils.py
index 98f0891..27cb13d 100644
--- a/repos/system_upgrade/common/libraries/utils.py
+++ b/repos/system_upgrade/common/libraries/utils.py
@@ -5,7 +5,7 @@ import six
 
 from leapp.exceptions import StopActorExecutionError
 from leapp.libraries.common import mounting
-from leapp.libraries.stdlib import STDOUT, CalledProcessError, api, config, run
+from leapp.libraries.stdlib import api, CalledProcessError, config, run, STDOUT
 
 
 def parse_config(cfg=None, strict=True):
diff --git a/repos/system_upgrade/common/models/selinux.py b/repos/system_upgrade/common/models/selinux.py
index 27c01ea..3305691 100644
--- a/repos/system_upgrade/common/models/selinux.py
+++ b/repos/system_upgrade/common/models/selinux.py
@@ -1,4 +1,4 @@
-from leapp.models import Model, fields
+from leapp.models import fields, Model
 from leapp.topics import SystemInfoTopic, TransactionTopic
```

File: Makefile (hunk 1 above)
Comment (fernflower) [reply Makefile]: Added a lint_fix target.
Also switched from git diff master..HEAD to git diff master this should catch problems even if files that are changed but not committed yet.

**Comment by fernflower** [comment]
leapp ci-build

**Comment by fernflower** [comment]
Okay, final run of `make lint_fix`:
```
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
index e0ae41b..7eca815 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/actor.py
@@ -3,11 +3,11 @@ import shutil
 
 from leapp import reporting
 from leapp.actors import Actor
-from leapp.models import SELinuxModules, SELinuxCustom
-from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
 from leapp.libraries.actor import selinuxapplycustom
 from leapp.libraries.actor.selinuxapplycustom import BACKUP_DIRECTORY
-from leapp.libraries.stdlib import run, CalledProcessError
+from leapp.libraries.stdlib import CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxModules
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
 
 WORKING_DIRECTORY = '/tmp/selinux/'
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
index 95c06bf..887f381 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/libraries/selinuxapplycustom.py
@@ -1,9 +1,8 @@
-import re
 import os
+import re
 import shutil
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
-
+from leapp.libraries.stdlib import api, CalledProcessError, run
 
 BACKUP_DIRECTORY = "/tmp/leapp-selinux/"
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
index b0ed347..0171f58 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxapplycustom/tests/component_test_selinuxapplycustom.py
@@ -2,8 +2,8 @@ import os
 
 import pytest
 
-from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxFacts, SELinuxModule, SELinuxModules, SELinuxRequestRPMs
 from leapp.reporting import Report
 
 TEST_MODULES = [
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
index b764a51..23bc9c2 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/actor.py
@@ -1,7 +1,7 @@
 from leapp.actors import Actor
-from leapp.models import SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs, RpmTransactionTasks
-from leapp.tags import FactsPhaseTag, IPUWorkflowTag
 from leapp.libraries.actor import selinuxcontentscanner
+from leapp.models import RpmTransactionTasks, SELinuxCustom, SELinuxFacts, SELinuxModules, SELinuxRequestRPMs
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
 
 
 class SELinuxContentScanner(Actor):
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
index e2f3de8..d512803 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/libraries/selinuxcontentscanner.py
@@ -2,8 +2,8 @@ import os
 import re
 from shutil import rmtree
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
 from leapp.libraries.common.config import version
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule
 
 # types and attributes that where removed between RHEL 7 and 8
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
index 2a98aa4..956277b 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxcontentscanner/tests/component_test_selinuxcontentscanner.py
@@ -2,11 +2,11 @@ import os
 
 import pytest
 
-from leapp.snactor.fixture import current_actor_context
-from leapp.models import SELinuxModule, SELinuxModules, SELinuxCustom, SELinuxFacts, SELinuxRequestRPMs
 from leapp.libraries.common.config import mock_configs
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
+from leapp.models import SELinuxCustom, SELinuxFacts, SELinuxModule, SELinuxModules, SELinuxRequestRPMs
 from leapp.reporting import Report
+from leapp.snactor.fixture import current_actor_context
 
 # compat module ensures compatibility with newer systems and is not part of testing
 TEST_MODULES = [
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
index 0ab2623..96e49b4 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/libraries/selinuxprepare.py
@@ -1,4 +1,4 @@
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModules
 
 
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
index 078ee76..105dcb2 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/component_test_selinuxprepare.py
@@ -2,12 +2,11 @@ import os
 
 import pytest
 
-from leapp.libraries.stdlib import api, run, CalledProcessError
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule, SELinuxModules
 from leapp.reporting import Report
 from leapp.snactor.fixture import current_actor_context
 
-
 TEST_MODULES = [
     ["400", "mock1"],
     ["99", "mock1"],
diff --git a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
index beeaf3e..0a6f4c9 100644
--- a/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
+++ b/repos/system_upgrade/common/actors/selinux/selinuxprepare/tests/unit_test_selinuxprepare.py
@@ -1,5 +1,5 @@
 from leapp.libraries.actor import selinuxprepare
-from leapp.libraries.stdlib import CalledProcessError, api, run
+from leapp.libraries.stdlib import api, CalledProcessError, run
 from leapp.models import SELinuxModule, SELinuxModules
 
 
diff --git a/repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py b/repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py
index 7094a75..b173f0e 100644
--- a/repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py
+++ b/repos/system_upgrade/common/actors/storagescanner/libraries/storagescanner.py
@@ -1,12 +1,20 @@
+import functools
 import os
 import subprocess
-import functools
-
 
-from leapp.models import StorageInfo, PartitionEntry, FstabEntry, MountEntry, LsblkEntry, \
-    PvsEntry, VgsEntry, LvdisplayEntry, SystemdMountEntry
 from leapp import reporting
 from leapp.libraries.stdlib import api
+from leapp.models import (
+    FstabEntry,
+    LsblkEntry,
+    LvdisplayEntry,
+    MountEntry,
+    PartitionEntry,
+    PvsEntry,
+    StorageInfo,
+    SystemdMountEntry,
+    VgsEntry
+)
 
 
 def aslist(f):
diff --git a/repos/system_upgrade/common/actors/storagescanner/tests/unit_test_storagescanner.py b/repos/system_upgrade/common/actors/storagescanner/tests/unit_test_storagescanner.py
index 3d942ec..7ee0c7f 100644
--- a/repos/system_upgrade/common/actors/storagescanner/tests/unit_test_storagescanner.py
+++ b/repos/system_upgrade/common/actors/storagescanner/tests/unit_test_storagescanner.py
@@ -4,8 +4,16 @@ from leapp import reporting
 from leapp.libraries.actor import storagescanner
 from leapp.libraries.common.testutils import create_report_mocked, logger_mocked
 from leapp.libraries.stdlib import api
-from leapp.models import (FstabEntry, LsblkEntry, LvdisplayEntry, MountEntry, PartitionEntry, PvsEntry,
-                          SystemdMountEntry, VgsEntry, )
+from leapp.models import (
+    FstabEntry,
+    LsblkEntry,
+    LvdisplayEntry,
+    MountEntry,
+    PartitionEntry,
+    PvsEntry,
+    SystemdMountEntry,
+    VgsEntry
+)
 
 CUR_DIR = os.path.dirname(os.path.abspath(__file__))
 
diff --git a/repos/system_upgrade/common/libraries/utils.py b/repos/system_upgrade/common/libraries/utils.py
index 98f0891..27cb13d 100644
--- a/repos/system_upgrade/common/libraries/utils.py
+++ b/repos/system_upgrade/common/libraries/utils.py
@@ -5,7 +5,7 @@ import six
 
 from leapp.exceptions import StopActorExecutionError
 from leapp.libraries.common import mounting
-from leapp.libraries.stdlib import STDOUT, CalledProcessError, api, config, run
+from leapp.libraries.stdlib import api, CalledProcessError, config, run, STDOUT
 
 
 def parse_config(cfg=None, strict=True):
diff --git a/repos/system_upgrade/common/models/selinux.py b/repos/system_upgrade/common/models/selinux.py
index 27c01ea..3305691 100644
--- a/repos/system_upgrade/common/models/selinux.py
+++ b/repos/system_upgrade/common/models/selinux.py
@@ -1,4 +1,4 @@
-from leapp.models import Model, fields
+from leapp.models import fields, Model
 from leapp.topics import SystemInfoTopic, TransactionTopic
```

The subsequent run of `make lint` finished with 0 return code.

File: Makefile [hunk 2]
```diff
@@ -106,6 +106,7 @@ help:
 	@echo "  install-deps-fedora    create python virtualenv and install there"
 	@echo "                         leapp-repository with dependencies for Fedora OS"
 	@echo "  lint                   lint source code"
+	@echo "  lint_fix               lint source code and fix isort violations inplace"
```
Comment (vinzenz) [inline Makefile:109]:
```suggestion
	@echo "  lint_fix               fixes isort violations inplace"
```

**Review by vinzenz — APPROVED** [review]

**Comment by fernflower** [comment]
leapp ci-build

File: Makefile [hunk 3]
```diff
@@ -259,6 +260,14 @@ lint:
 		bash -c "[[ ! -z '$${SEARCH_PATH}' ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint --py3k" && \
 		echo "--- Linting done. ---"; \
 	fi
+	. $(VENVNAME)/bin/activate; \
+	git diff $(MASTER_BRANCH) --name-only | xargs isort -c --diff && \
+	echo "--- isort check done. ---;"
```
Comment (pirat89) [inline Makefile:265]: I suggest to update this part to inform people about `make lint_fix`. E.g. like that:
```diff
diff --git a/Makefile b/Makefile
index c4d82dc7..64e56008 100644
--- a/Makefile
+++ b/Makefile
@@ -260,9 +260,16 @@ lint:
 		bash -c "[[ ! -z '$${SEARCH_PATH}' ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint --py3k" && \
 		echo "--- Linting done. ---"; \
 	fi
+
 	. $(VENVNAME)/bin/activate; \
-	git diff $(MASTER_BRANCH) --name-only | xargs isort -c --diff && \
-	echo "--- isort check done. ---;"
+	git diff $(MASTER_BRANCH) --name-only | xargs isort -c --diff || \
+	{ \
+		echo; \
+		echo "------------------------------------------------------------------------------" \
+		echo "Hint: Apply the required changes."; \
+		echo "      Execute the following command to apply them automatically: make lint_fix"; \
+		exit 1; \
+	} && echo "--- isort check done. ---";
 
 lint_fix:
 	. $(VENVNAME)/bin/activate; \
```


The output looks then something like this:
```
ERROR: /home/pstodulk/special/leapp/leapp-repository/repos/system_upgrade/el7toel8/actors/checkgrep/actor.py Imports are incorrectly sorted.
--- /home/pstodulk/special/leapp/leapp-repository/repos/system_upgrade/el7toel8/actors/checkgrep/actor.py:before	2021-12-02 10:39:32.098772
+++ /home/pstodulk/special/leapp/leapp-repository/repos/system_upgrade/el7toel8/actors/checkgrep/actor.py:after	2021-12-02 10:59:27.462197
@@ -1,9 +1,9 @@
+from leapp import reporting
 from leapp.actors import Actor
 from leapp.libraries.common.rpms import has_package
 from leapp.models import InstalledRedHatSignedRPM
-from leapp.reporting import Report, create_report
+from leapp.reporting import create_report, Report
 from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
-from leapp import reporting
 
 
 class CheckGrep(Actor):

------------------------------------------------------------------------------ echo Hint: Apply the required changes.
      Execute the following command to apply them automatically: make lint_fix
make: *** [Makefile:249: lint] Error 1
```

Comment (fernflower) [reply Makefile]: Applied, thanks!

**Comment by fernflower** [comment]
@pirat89 @vinzenz Waiting for tests

File: Makefile [hunk 4]
```diff
@@ -260,6 +261,21 @@ lint:
 		echo "--- Linting done. ---"; \
 	fi
 
+	. $(VENVNAME)/bin/activate; \
+	git diff $(MASTER_BRANCH) --name-only | xargs isort -c --diff || \
+	{ \
+		echo; \
+		echo "------------------------------------------------------------------------------" \
```
Comment (pirat89) [inline Makefile:268]:
```suggestion
		echo "------------------------------------------------------------------------------"; \
```

**Review by pirat89 — APPROVED** [review]

**Pull request merged**